src/js/templates/error.ts

```diff
@@ -1,6 +1,6 @@
 import type { HTMLTemplateString } from 'types/generic.type';
 
-export default (id: string, message: string, errorCode: string, code: string): HTMLTemplateString => {
+export default (id: string, message: string | null, errorCode: string, code: string): HTMLTemplateString => {
```

Comment:
hmm this is an interesting case - can you use the optional `?` and then have required params after? prob not right? if so good as is

Reply:
yeah. can't have optional params before required params.